### PR TITLE
New Instrument Cal CSVs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Included are instrument cal csvs for instruments returning from vendor rehab (I mean refurb!)
Also included are a few random NUTNRB, NUTNRM and NUTNRN pre-deploy and/or recovery csvs.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
Update - I deleted 2 NUTNR cal csvs from this pull request (both for S/N 1107).  They were already submitted as part of the Irminger 6 cal csv pull request #49 and duplicated in error.

**Comment by rtravis-whoi** [comment]
Update - deleted duplicate OPTAA files that are also in PR #41.  (CGINS-OPTAAD-00256__20181204.csv and associated taarray/tcarray files).

**Comment by rtravis-whoi** [comment]
Added a new OPTAA cal for incoming OPTAA-D s/n 207

**Comment by rtravis-whoi** [comment]
Added 2 new NUTNRB cal csvs from recovered NUTNR testing (recovered on Irminger 6)

File: calibration/OPTAAD/CGINS-OPTAAD-00209__20190812.csv [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+ACS-209,CC_tbins,"[1.339284, 2.414694, 3.439231, 4.489000, 5.554286, 6.476923, 7.466364, 8.499600, 9.485455, 10.480000, 11.492083, 12.510800, 13.492609, 14.526000, 15.492917, 16.467200, 17.498333, 18.488696, 19.476923, 20.477200, 21.482500, 22.480800, 23.491200, 24.474000, 25.492222, 26.505200, 27.477600, 28.494615, 29.499231, 30.504231, 31.512414, 32.457407, 33.554444, 34.443939]",
+ACS-209,CC_taarray,SheetRef:CC_taarray,
+ACS-209,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00209__20190812.csv]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00222__20190722.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ACS-222,CC_tbins,"[0.981579, 1.151934, 2.471600, 3.450000, 4.499412, 5.493462, 6.491200, 7.474400, 8.497500, 9.483600, 10.476522, 11.470000, 12.492083, 13.488929, 14.506250, 15.508333, 16.502800, 17.482174, 18.485417, 19.487200, 20.467917, 21.499583, 22.473704, 23.510833, 24.489600, 25.506667, 26.479583, 27.478000, 28.494000, 29.467917, 30.504815, 31.495185, 32.516087, 33.527742, 34.450000]",
+ACS-222,CC_taarray,SheetRef:CC_taarray,
+ACS-222,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00222__20190722.csv]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00257__20190614.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+ACS-257,CC_tbins,"[1.643781, 2.286364, 3.449062, 4.450435, 5.513125, 6.523333, 7.512800, 8.520000, 9.510400, 10.520800, 11.490000, 12.478077, 13.495600, 14.494800, 15.478000, 16.478800, 17.492800, 18.507692, 19.507083, 20.499200, 21.521154, 22.494000, 23.492000, 24.489583, 25.501481, 26.468400, 27.497308, 28.489200, 29.475556, 30.513846, 31.488077, 32.488519, 33.510000, 34.512581]",
+ACS-257,CC_taarray,SheetRef:CC_taarray,
+ACS-257,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00257__20190614.csv]: Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0050__20190709.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0050,CC_cala,0.0454,source_file PCO2W-B_SAMI-CO2_SN_C0050_Calibration_Files_2019-07-09.zip > C050_070519ed_CalCert.pdf
+C0050,CC_calb,0.6566,
+C0050,CC_calc,-1.5455,
+C0050,CC_calt,7.5864,
+C0050,CC_ea434,19706,constant for each batch of reagent
+C0050,CC_ea620,34,constant for each batch of reagent
+C0050,CC_eb434,3073,constant for each batch of reagent
+C0050,CC_eb620,44327,constant for each batch of reagent
```
Comment (reedan88) [inline calibration/PCO2WB/CGINS-PCO2WB-C0050__20190709.csv]: Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0069__20190805.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0069,CC_cala,0.0986,source_file PCO2W-B_SAMI-CO2_SN_C0069_Calibration_Certificate_2019-08-05.pdf
+C0069,CC_calb,0.3666,
+C0069,CC_calc,-1.2383,
+C0069,CC_calt,13.5463,
+C0069,CC_ea434,19706,constant
+C0069,CC_ea620,34,constant
+C0069,CC_eb434,3073,constant
+C0069,CC_eb620,44327,constant
```
Comment (reedan88) [inline calibration/PCO2WB/CGINS-PCO2WB-C0069__20190805.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0156__20190627.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0156,CC_ea434,18103.0,source_file PHSEN-D_SAMI-pH_SN_P0156_Validation_Certificate_2019-06-27.pdf
+P0156,CC_ea578,97.8,
+P0156,CC_eb434,2296.0,
+P0156,CC_eb578,40427.0,
+P0156,CC_ind_off,0,
+P0156,CC_ind_slp,1,
+P0156,CC_psal,35,Constant
```
Comment (reedan88) [inline calibration/PHSEND/CGINS-PHSEND-P0156__20190627.csv]: Looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00269__20190711.csv [hunk 7]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+269,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347] ", source_file SPKIR-B_OCR-507_SN_269_Calibration_Files_2019-07-11.zip > DI7269C.cal
+269,CC_offset,"[2147757211.7, 2148311470.6, 2147513903.3, 2146916672, 2146986365.5, 2146569068.8, 2147637121.1] ",
+269,CC_scale,"[0.000000211815651336, 0.000000206745203989, 0.00000020567319826, 0.000000208430175114, 0.000000219956989146, 0.000000218069540674, 0.000000213696530841] ",
```
Comment (reedan88) [inline calibration/SPKIRB/CGINS-SPKIRB-00269__20190711.csv]: Looks good.

File: calibration/SPKIRB/CGINS-SPKIRB-00272__20190711.csv [hunk 8]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+272,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347] ", source_file SPKIR-B_OCR-507_SN_272_Calibration_Files_2019-07-11.zip > DI7272D.cal
+272,CC_offset,"[2147762779.9, 2148114030.1, 2147393933.9, 2147046283.8, 2147122694.4, 2147431033.9, 2147725918.4] ",
+272,CC_scale,"[0.000000216905350231, 0.000000205763879597, 0.0000002071381884, 0.00000020075458698, 0.000000218857490668, 0.000000209188322547, 0.000000223834020508] ",
```
Comment (reedan88) [inline calibration/SPKIRB/CGINS-SPKIRB-00272__20190711.csv]: Looks good.

File: calibration/SPKIRB/CGINS-SPKIRB-00273__20190711.csv [hunk 9]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+273,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347] ", source_file SPKIR-B_OCR-507_SN_273_Calibration_Files_2019-07-11.zip > DI7273D.cal
+273,CC_offset,"[2147435914, 2147314725.5, 2146994203.5, 2147301712.4, 2147951020.7, 2147830833.3, 2146940810] ",
+273,CC_scale,"[0.000000196192790336, 0.000000196580668577, 0.000000205363748175, 0.000000200638593902, 0.000000210902843331, 0.000000213910229621, 0.000000207762116442] ",
```
Comment (reedan88) [inline calibration/SPKIRB/CGINS-SPKIRB-00273__20190711.csv]: Looks good.

File: calibration/SPKIRB/CGINS-SPKIRB-00289__20190709.csv [hunk 10]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+289,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347] ",source_file SPKIR-B_OCR-507_SN_289_Calibration_Files_2019-07-09.zip > DI7289E.cal
+289,CC_offset,"[2147031821.2, 2147192886.8, 2148098839.8, 2147682094.6, 2147276813.9, 2147624525.9, 2147375093] ",
+289,CC_scale,"[0.000000208145481567, 0.00000020431922929, 0.000000204285195835, 0.000000215366417178, 0.000000192597164295, 0.000000219269271436, 0.000000206519441507] ",
```
Comment (reedan88) [inline calibration/SPKIRB/CGINS-SPKIRB-00289__20190709.csv]: Looks good.

File: calibration/ADCPTF/CGINS-ADCPTF-18594__20190808.csv [hunk 11]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+18594,CC_scale_factor1,0.45,source_file ADCPT-F_WHS150-1500_SN_18594_Configuration_Files_2019-08-08.zip
+18594,CC_scale_factor2,0.45,
+18594,CC_scale_factor3,0.45,
+18594,CC_scale_factor4,0.45,
```
Comment (reedan88) [inline calibration/ADCPTF/CGINS-ADCPTF-18594__20190808.csv]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00129__20190812__CC_taarray.ext [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+-0.012811,-0.009385,-0.00739,-0.006454,-0.005996,-0.0055,-0.005103,-0.004644,-0.004011,-0.00355,-0.003165,-0.002623,-0.002136,-0.001745,-0.001349,-0.001106,-0.000882,-0.00073,-0.000529,-0.000409,-0.000235,-0.000184,-9.1e-05,0,-6e-06,-9e-06,2.5e-05,9e-05,0.000224,0.000191,0.000205,0.00023,0.000187,0.000203
+-0.013029,-0.009534,-0.007478,-0.006579,-0.006085,-0.005581,-0.00523,-0.004754,-0.004071,-0.003632,-0.003197,-0.002743,-0.002178,-0.001833,-0.001465,-0.001216,-0.000955,-0.000771,-0.000552,-0.000441,-0.000231,-0.000189,-6.4e-05,0,2e-06,5e-06,1.8e-05,0.000144,0.000271,0.000147,0.000234,0.000247,0.000242,0.00041
+-0.012707,-0.009264,-0.007277,-0.006388,-0.005868,-0.005342,-0.005006,-0.004611,-0.003954,-0.003493,-0.003112,-0.002626,-0.00209,-0.001799,-0.001395,-0.001143,-0.00092,-0.000754,-0.000509,-0.000379,-0.000188,-0.000164,-8.2e-05,0,3.7e-05,3.7e-05,3.5e-05,0.00019,0.000265,0.000198,0.000297,0.000259,0.000218,0.000441
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00129__20190812__CC_taarray.ext]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00129__20190812__CC_tcarray.ext [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+0.012301,0.016595,0.018367,0.017651,0.016635,0.015542,0.014065,0.013197,0.01226,0.011399,0.010503,0.00981,0.008879,0.00782,0.006977,0.006256,0.005509,0.004911,0.003942,0.003244,0.002352,0.00162,0.000839,0,-0.000959,-0.001873,-0.002807,-0.003645,-0.004583,-0.005432,-0.006267,-0.007238,-0.008077,-0.008755
+0.012596,0.017125,0.018838,0.018073,0.017007,0.015907,0.014641,0.01374,0.01275,0.012043,0.010932,0.010009,0.009157,0.00842,0.007379,0.006556,0.005737,0.004998,0.00424,0.003409,0.002449,0.001773,0.000935,0,-0.000942,-0.001812,-0.002849,-0.003699,-0.004706,-0.005517,-0.00635,-0.007384,-0.008327,-0.008679
+0.01373,0.018153,0.019699,0.01892,0.017918,0.01683,0.015481,0.014544,0.013506,0.012699,0.011686,0.010747,0.009871,0.008853,0.007709,0.00696,0.006198,0.005502,0.004327,0.003595,0.002746,0.001904,0.00102,0,-0.000917,-0.001766,-0.00283,-0.003687,-0.004778,-0.005678,-0.006562,-0.007669,-0.008732,-0.009339
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00129__20190812__CC_tcarray.ext]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00207__20190725.csv [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+ACS-207,CC_tbins,"[1.341611, 2.429778, 3.460741, 4.486190, 5.561500, 6.470000, 7.492609, 8.484167, 9.509583, 10.498333, 11.493750, 12.482917, 13.471304, 14.485926, 15.509565, 16.489200, 17.490000, 18.501600, 19.491250, 20.498846, 21.493600, 22.506923, 23.498750, 24.504800, 25.506000, 26.479200, 27.489231, 28.497600, 29.506429, 30.503462, 31.490385, 32.501852, 33.524643, 34.468929, 35.000000]",
+ACS-207,CC_taarray,SheetRef:CC_taarray,
+ACS-207,CC_tcarray,SheetRef:CC_tcarray,
```
[the remark this answers is not in the document]
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00207__20190725.csv]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00207__20190725__CC_taarray.ext [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+0.004487,0.00462,0.004411,0.003972,0.003685,0.003528,0.00315,0.002904,0.002611,0.002293,0.002096,0.00177,0.001581,0.001386,0.001225,0.00096,0.00086,0.000747,0.000491,0.000437,0.000345,0.000241,0.000144,4.1e-05,0,-0.000166,-0.00024,-0.000318,-0.000457,-0.00047,-0.000712,-0.000683,-0.000855,-0.000925,-0.000946
+0.004279,0.004508,0.004254,0.003911,0.003618,0.003397,0.003062,0.002816,0.002499,0.002267,0.002,0.001661,0.001535,0.001319,0.001182,0.000924,0.000818,0.000658,0.00044,0.000384,0.000363,0.000204,0.000119,0.000119,0,-0.000186,-0.000232,-0.000274,-0.000435,-0.000388,-0.000663,-0.000608,-0.000869,-0.00086,-0.00091
+0.004246,0.004463,0.004252,0.003919,0.003519,0.003374,0.003073,0.00279,0.002537,0.002242,0.002,0.001665,0.001539,0.001293,0.001107,0.000866,0.00078,0.00061,0.000331,0.00032,0.000289,0.000155,4.9e-05,2.2e-05,0,-0.000164,-0.000297,-0.000275,-0.000427,-0.000456,-0.000657,-0.000726,-0.000938,-0.000894,-0.001118
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00207__20190725__CC_taarray.ext]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00207__20190725__CC_tcarray.ext [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+0.060444,0.055262,0.048632,0.042682,0.037916,0.033249,0.029192,0.025874,0.023011,0.020376,0.018301,0.016652,0.014762,0.01294,0.011147,0.009665,0.008223,0.006859,0.005703,0.004473,0.003503,0.002495,0.001655,0.000759,0,-0.00072,-0.001362,-0.001959,-0.002559,-0.003022,-0.003315,-0.003689,-0.004099,-0.004381,-0.004544
+0.061128,0.055797,0.049166,0.043291,0.038377,0.033514,0.029449,0.026134,0.023185,0.020561,0.0186,0.01683,0.015061,0.013308,0.011448,0.009895,0.008408,0.006994,0.005652,0.004535,0.003419,0.002543,0.001645,0.000776,0,-0.000666,-0.001393,-0.001936,-0.002503,-0.002988,-0.003341,-0.003678,-0.004141,-0.004434,-0.004682
+0.06157,0.0562,0.049514,0.043527,0.038734,0.034045,0.029937,0.026588,0.023631,0.020948,0.018811,0.017049,0.015234,0.013334,0.011581,0.009977,0.008413,0.007031,0.005738,0.004551,0.003546,0.002446,0.001581,0.0007,0,-0.00078,-0.001348,-0.001933,-0.002553,-0.002998,-0.003316,-0.003671,-0.003984,-0.004256,-0.004335
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00207__20190725__CC_tcarray.ext]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00209__20190812.csv (hunk 1 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00209__20190812.csv]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00209__20190812__CC_taarray.ext [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+-0.005871,-0.004471,-0.004054,-0.003864,-0.003655,-0.003469,-0.003212,-0.003027,-0.002762,-0.002591,-0.002294,-0.002097,-0.001902,-0.001677,-0.001384,-0.001193,-0.000905,-0.000771,-0.000538,-0.000411,-0.000254,-0.000117,-0.000137,4.4e-05,0,9.9e-05,8.5e-05,0.000179,0.000212,0.000178,0.000182,0.0004,0.000284,0.000248
+-0.005947,-0.004556,-0.004092,-0.003956,-0.003758,-0.003545,-0.003285,-0.003112,-0.002805,-0.002637,-0.002382,-0.002153,-0.001976,-0.001685,-0.001399,-0.001225,-0.000934,-0.000732,-0.000511,-0.000419,-0.00021,-0.00011,-0.000128,2.3e-05,0,6.9e-05,9.9e-05,9.8e-05,0.000226,0.000143,0.000178,0.000376,0.000252,0.000208
+-0.006243,-0.004739,-0.004286,-0.004108,-0.003872,-0.003648,-0.003343,-0.003165,-0.002846,-0.002734,-0.002365,-0.002195,-0.001984,-0.001669,-0.00134,-0.001208,-0.00092,-0.000711,-0.000504,-0.00038,-0.000165,-4.9e-05,-0.000106,8e-05,0,0.00015,0.000152,0.000137,0.000208,0.000164,0.000224,0.000379,0.000245,0.000275
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00209__20190812__CC_taarray.ext]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00209__20190812__CC_tcarray.ext [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+0.009893,0.010423,0.009903,0.009116,0.008525,0.007904,0.007205,0.006746,0.006097,0.005614,0.005107,0.004631,0.003987,0.003428,0.002982,0.00248,0.002103,0.001674,0.001451,0.001248,0.000904,0.000615,0.000431,0.00021,0,-0.000132,-0.00035,-0.000415,-0.000585,-0.000535,-0.000718,-0.001065,-0.001673,-0.002585
+0.010291,0.010831,0.010331,0.009622,0.008984,0.008401,0.007753,0.007226,0.006643,0.006129,0.005661,0.005029,0.004373,0.0038,0.003294,0.002773,0.002354,0.001979,0.001609,0.001352,0.001032,0.000715,0.000526,0.000161,0,-0.000165,-0.000299,-0.000436,-0.000636,-0.000616,-0.000816,-0.001193,-0.001724,-0.002597
+0.010612,0.01099,0.010582,0.009908,0.009147,0.008431,0.007804,0.007167,0.00658,0.006088,0.005527,0.004892,0.004231,0.003748,0.003131,0.002596,0.002259,0.001764,0.001551,0.00129,0.001044,0.000687,0.000474,0.000187,0,-7.8e-05,-0.00031,-0.000351,-0.000516,-0.000559,-0.000725,-0.00105,-0.00164,-0.002552
```
Comment (cdobs) [inline calibration/OPTAAD/CGINS-OPTAAD-00209__20190812__CC_tcarray.ext]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00222__20190722.csv (hunk 2 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00222__20190722.csv]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00257__20190614.csv (hunk 3 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00257__20190614.csv]: Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0050__20190709.csv (hunk 4 above)
Comment (cdobs) [reply calibration/PCO2WB/CGINS-PCO2WB-C0050__20190709.csv]: Looks good.

File: calibration/PCO2WB/CGINS-PCO2WB-C0069__20190805.csv (hunk 5 above)
Comment (cdobs) [reply calibration/PCO2WB/CGINS-PCO2WB-C0069__20190805.csv]: Looks good.

File: calibration/PHSEND/CGINS-PHSEND-P0156__20190627.csv (hunk 6 above)
Comment (cdobs) [reply calibration/PHSEND/CGINS-PHSEND-P0156__20190627.csv]: Looks good.

File: calibration/SPKIRB/CGINS-SPKIRB-00272__20190711.csv (hunk 8 above)
Comment (cdobs) [reply calibration/SPKIRB/CGINS-SPKIRB-00272__20190711.csv]: Looks good.

File: calibration/SPKIRB/CGINS-SPKIRB-00269__20190711.csv (hunk 7 above)
Comment (cdobs) [reply calibration/SPKIRB/CGINS-SPKIRB-00269__20190711.csv]: Looks good.

File: calibration/SPKIRB/CGINS-SPKIRB-00273__20190711.csv (hunk 9 above)
Comment (cdobs) [reply calibration/SPKIRB/CGINS-SPKIRB-00273__20190711.csv]: Looks good.

File: calibration/SPKIRB/CGINS-SPKIRB-00289__20190709.csv (hunk 10 above)
Comment (cdobs) [reply calibration/SPKIRB/CGINS-SPKIRB-00289__20190709.csv]: Looks good.

**Pull request merged**